### PR TITLE
FIX Kutjevo Botany APC that can't be fixed

### PR DESCRIPTION
**Pull request by majchroo**
# About the pull request

**TL;DR APC can't be fixed, moved to a place where it can be fixed.**

Kutjevo Botany APC is placed on a "dirt" tile (auto_turf to be precise) and so it means that engineers can't use crowbar on that tile and rewire the terminal underneath.

# Explain why it's good for the game

Engineers will be able to fix power in botany on Kutjevo.

# Testing Photographs and Procedure
Moved the APC using StrongDMM. Booted up the server and made sure I can fix it as a spawned engineer.

<details>
<summary>Screenshots & Videos</summary>

Before:
image

After:
image

Proof of testing:
image

</details>

# Changelog

:cl:Adamix147
maptweak: Moved the unfixable APC at Kutjevo Botany to make it fixable again.
/:cl:

### CODE REVIEW

File: html/changelogs/majchroo-pr-6909.yml
Comment (Drulikar) [inline html/changelogs/majchroo-pr-6909.yml:1]: Please remove this file. It will be generated automatically on merge

**Review by Drulikar — CHANGES_REQUESTED** [review]

File: html/changelogs/majchroo-pr-6909.yml
Comment (majchroo) [reply html/changelogs/majchroo-pr-6909.yml:1]: File removed in commit 157b26d

**Review by Drulikar — APPROVED** [review]
